### PR TITLE
Jr/https

### PR DESCRIPTION
**Pull request by clubzspadez**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
     <link rel="stylesheet" type="text/css" href="styleBlank.css">
     <link rel="stylesheet" type="text/css" href="style.css">
-    <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
-    <link href='http://fonts.googleapis.com/css?family=Open+Sans:400;300' rel='stylesheet' type='text/css'>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
```
Comment (chakrm) [inline index.html]: this looks better now that you made it https

File: index.html [hunk 2]
```diff
@@ -6,8 +6,8 @@
...
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400;300' rel='stylesheet' type='text/css'>
```
Comment (chakrm) [inline index.html]: same thing

File: index.html (hunk 1 above)
Comment (clubzspadez) [inline index.html]: ok thank you 👍

**Comment by chakrm** [comment]
this update looks good now that it has https

**Comment by wingchi** [comment]
:shipit:

File: index.html (hunk 1 above)
Comment (clubzspadez) [inline index.html]: :gun:

**Comment by chakrm** [comment]
:100:  :+1:

**Pull request merged**